### DISCUSSION TITLE
Custom JS

### DISCUSSION THREAD

**Discussion by mjabdulm** [zoidsh/meru#319] [Feature Requests]
Hi,

Is there a way to provide a custom JS to the app?

**Comment by timche** [comment]
There isn't a way at the moment, but this could be potentially added in the future.

**Comment by mjabdulm** [comment]
Thanks a lot for you response. I’m looking forward to it.

**Comment by serhumanos** [comment]
Thanks! I’m looking forward too for this feature for who used to use tampermonkey/greasemonkey scripts
